### ISSUE TITLE
Separate Damage Scaling Multiplier in Config

### ISSUE THREAD

**Issue by Boolyman** [Growlith1223/ArsMagica2#314]
Hello, it looks like you have your hands full with bugs, and most of mine have been covered already.  Hopefully a fix is coming soon for some of these. 

While I am here though, I might as well put in my humble request for a quality of life change.  Instead of having a single Damage Multiplier in the config file, that scales up ALL magic damage done, regardless of whether it is a mob or a player, can you please separate the two?  When using mods like Infernal Mobs or Scaling Health, the mobs have increased defenses, while the player does not usually...  Modpacks like this make Ars Magica nearly unusable because the low level spells only hit mobs for a measly 15 damage... while the same damage done to a player is devastating.  Yes, more complicated spell combinations can do much more damage... but the player will never make it to a point where their mana pool can handle these spells, if they have to grind up using spells that don't even scratch the mobs.

Long Story Short:  Can you please give us a config option to scale up or down ONLY damage done by player spells?

**Comment by bookerthegeek** [comment]
Actually seperate configs would be welcome as I am making it harder on my pkayers so being avle to make mods hit harder would be nice without also overpowering the player at the same time.

**Comment by xenxero1** [comment]
Just gonna push the envelope here. I'd greatly appreciate seperate damage modifiers for each magic damage type. Or some way to make it more configurable. I'd like to make lightning stronger (potentially during weather events), and reduce the damage basic magic does. Let's face it, Mana Blast is OP as hell.
